lib/param_parsing/environment_test.go

```diff
@@ -7,6 +7,19 @@ import (
 	"github.com/warrensbox/terraform-switcher/lib"
 )
 
+func TestGetParamsFromEnvironment_arch_from_env(t *testing.T) {
+	logger = lib.InitLogger("DEBUG")
+	var params Params
+	expected := "amd64_from_env"
+	_ = os.Setenv("TF_ARCH", expected)
+	params = initParams(params)
+	params = GetParamsFromEnvironment(params)
+	_ = os.Unsetenv("TF_ARCH")
```

Comment:
Maybe one for another time, but should this unset be deferred after the initial setting? This might avoid tests interferring with each other if this one fails before it is unset

Reply:
This test is copy-pasta from the one right down the code. So I'd leave this for another time please =)